feature/bgp/policybase/otg_tests/comm_match_action_test/comm_match_action_test.go

```diff
@@ -0,0 +1,295 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package comm_match_action_test
+
+import (
+	"testing"
+	"time"
+
+	"google3/third_party/open_traffic_generator/gosnappi/gosnappi"
+	"google3/third_party/openconfig/featureprofiles/internal/cfgplugins/cfgplugins"
+	"google3/third_party/openconfig/featureprofiles/internal/deviations/deviations"
+	"google3/third_party/openconfig/featureprofiles/internal/fptest/fptest"
+	"google3/third_party/openconfig/featureprofiles/internal/otgutils/otgutils"
+	"google3/third_party/openconfig/ondatra/gnmi/gnmi"
+	"google3/third_party/openconfig/ondatra/gnmi/oc/oc"
+	"google3/third_party/openconfig/ondatra/ondatra"
+)
+
+const (
+	prefixV4Len  = 30
+	prefixV6Len  = 126
+	trafficPps   = 100
+	totalPackets = 1200
+	bgpName      = "BGP"
+)
+
+var prefixesV4 = [][]string{
+	{"198.51.100.0", "198.51.100.4"},
+	{"198.51.100.8", "198.51.100.12"},
+	{"198.51.100.16", "198.51.100.20"},
+	{"198.51.100.24", "198.51.100.28"},
+	{"198.51.100.32", "198.51.100.40"},
+}
+
+var prefixesV6 = [][]string{
+	{"2048:db1:64:64::0", "2048:db1:64:64::4"},
+	{"2048:db1:64:64::8", "2048:db1:64:64::12"},
+	{"2048:db1:64:64::16", "2048:db1:64:64::20"},
+	{"2048:db1:64:64::24", "2048:db1:64:64::28"},
+	{"2048:db1:64:64::32", "2048:db1:64:64::40"},
+}
+
+var matchStdCcomms = [][][]int{
+	{
+		{5, 5},
+	},
+}
+
+var addStdComms = [][][]int{
+	{
+		{10, 10}, {20, 20}, {30, 30},
+	},
+}
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func configureImportBGPPolicy(t *testing.T, dut *ondatra.DUTDevice, ipv4 string, ipv6 string) {
+
+	communitySetName := "add_std_comms"
+	// TODO: actions/bgp-actions/set-community/config/method = REFERENCE.
+	// At the moment no support for oc.BgpPolicy_SetCommunityMethodType_REFERENCE.
```

Comment:
If you mean a DUT doesn't support this, we should still implement this in the code.  DUT will fail and deviations can be added to workaround issue if there is some alternative way to configure the community list.  (for example, via INLINE)

Reply:
I meant irrespective of the DUT. and in general.

Reply:
`REFERENCE` should be in the code, but if that enum is not working,  try `INLINE` and see if that works instead of `REFERENCE`.  If `INLINE` works, then it can be used as a deviation to be removed later.